pkg/network/tracer/ebpf_conntracker.go

```diff
@@ -115,22 +112,22 @@ func NewEBPFConntracker(cfg *config.Config) (netlink.Conntracker, error) {
 
 func (e *ebpfConntracker) dumpInitialTables(ctx context.Context, cfg *config.Config) error {
 	e.consumer = netlink.NewConsumer(cfg.ProcRoot, cfg.ConntrackRateLimit, true)
-	e.decoder = netlink.NewDecoder()
 	defer e.consumer.Stop()
 
 	for _, family := range []uint8{unix.AF_INET, unix.AF_INET6} {
 		events, err := e.consumer.DumpTable(family)
```

Comment:
I think to gain maximum benefit from this PR, we should discard the data at an even lower level, so that we don't spend the cpu+memory on decoding all the netlink messages.